backend/test_observer/data_access/models.py

```diff
@@ -229,6 +239,11 @@ class TestExecution(Base):
     status: Mapped[TestExecutionStatus] = mapped_column(
         default=TestExecutionStatus.NOT_STARTED
     )
+    review_status: Mapped[list[TestExecutionReviewDecision]] = mapped_column(
+        ARRAY(Enum(TestExecutionReviewDecision)),
+        default=[],
+    )
```

Comment:
Again might be better if it's a set

Reply:
I think PostgreSQL doesn't have a SET data structure on the database level. Good balance I found is to define it as a set on FastAPI-level and ARRAY on PostgreSQL-level